node-tests/fixtures/controller-test/rfc232-unit.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
```

Comment:
since only a unit test is generated, should this be named `rfc232-default` or is the current name fine?

Reply:
as is seems fine to me, it's not lying -- it is a unit test.

Reply:
> it is a unit test

is it though... :trollface: 

I'd just use `rfc232.js` in this case, but it doesn't matter much.